js/index.js

```diff
@@ -0,0 +1,44 @@
+const cardsContainer = document.querySelector(".cards");
+
+const images = [
+  { id: 1, name: "graduated-emoji", url: "./images/front-pic1.png" },
+  { id: 2, name: "thinking-imoji", url: "./images/front-pic2.png" },
+  { id: 3, name: "crying-emoji", url: "./images/front-pic3.png" },
+  { id: 4, name: "drink-emoji", url: "./images/front-pic4.png" },
+  { id: 5, name: "winking-emoji", url: "./images/front-pic5.png" },
+  { id: 6, name: "dream-emoji", url: "./images/front-pic6.png" },
+  { id: 7, name: "wondering-emoji", url: "./images/front-pic7.png" },
+  { id: 8, name: "smile-emoji", url: "./images/front-pic8.png" },
+];
+
+const imagePickList = [...images, ...images];
+
+const cardCount = imagePickList.length;
+
+function buildCard(image) {
```

Comment:
You have this function, but this is the only function in the script so far. 
It could be a better practice to separate some other functionality into functions too. 
For example, you are calling this function to build a card, so there could also be functions to render all the cards (the loop that you have at the end), even the doubling and shuffling of the array of cards could be separate functions! 
That way the code would be more structured and a bit easier to read and you would also practise packing everything into separated things :)

Reply:
I made 2 other function that I will commit